src/opera/commands/info.py

```diff
@@ -0,0 +1,99 @@
+import argparse
+import json
+from os import path
+from pathlib import Path, PurePath
+
+import yaml
+
+from opera.error import DataError, ParseError
+from opera.parser import tosca
+from opera.storage import Storage
+
+
+def add_parser(subparsers):
+    parser = subparsers.add_parser(
+        "info",
+        help="Retrieve service template outputs"
```

Comment:
```suggestion
        help="Show information about the current project."
```

Maybe not project, but workspace? Or another word more suitable? Not sure here.

Reply:
I think project is alright.